README.md

```diff
@@ -10,17 +10,21 @@
 
 [Checkov](https://github.com/bridgecrewio/checkov) is a static code analysis tool for infrastrucutre-as-code. 
 
-The Checkov Extension for VSCODE enables developers to get real-time scan results, as well as inline fix suggestions as they develop cloud infratructure.
+The Checkov Extension for VSCODE enables developers to get real-time scan results, as well as inline fix suggestions as they develop cloud infratructure. 
 
-Features include:
+The extension is currently available for download direcrtly from the [Visual Studio Extension Marketplace](https://marketplace.visualstudio.com/items?itemName=Bridgecrew.checkov) and its source code is available in an [Apache 2.0 licensed repository](https://github.com/bridgecrewio/checkov-vscode). Development of the extension is ongoing and it is available for pre-release usage 🚧 .
 
-* 500 built-in policies covering security and compliance best practices for AWS, Azure and Google Cloud.
+Activating the extension requires submission of one-time Bridgecrew API Token that can be obtained by [creating a new Bridgecrew platform account](https://docs.bridgecrew.io/docs/get-api-token). It uses open [Bridgecrew Developer APIs](https://docs.bridgecrew.io/reference) to evaluate code and offer automated inline fixes. For more information about data shared with Bridgecrew see the [Disclaimer](#disclaimer) section below).
+
+Extension features include:
+
+* [500 built-in policies](https://github.com/bridgecrewio/checkov/blob/master/docs/3.Scans/resource-scans.md) covering security and compliance best practices for AWS, Azure and Google Cloud.
 * Terraform, Terraform Plan, CloudFormation, Kubernetes, Helm, Serverless and ARM template scanning.
-* Detects AWS credentials in EC2 Userdata, Lambda environment variables and Terraform providers.
-* In Terraform, checks support evaluation of arguments expressed in variables and remote modules to their actual values.
-* Supports inline suppression via comments.
+* Detects [AWS credentials](https://github.com/bridgecrewio/checkov/blob/master/docs/3.Scans/Credentials%20Scans.md) in EC2 Userdata, Lambda environment variables and Terraform providers.
+* In Terraform, checks support evaluation of arguments expressed in [variables](https://github.com/bridgecrewio/checkov/blob/master/docs/2.Concepts/Evaluations.md) and remote modules to their actual values.
+* Supports inline [suppression](https://github.com/bridgecrewio/checkov/blob/master/docs/2.Concepts/Suppressions.md) via comments.
 * Links to policy descriptions, rationales as well as step by step instructions for fixing known misconfigurations.
-*  Fix suggestions for commonly misconfigured Terraform and CloudFormation attributes.
+* Fix suggestions for commonly misconfigured Terraform and CloudFormation attributes.🪄 🪄 🪄 
```

Comment:
What are the symbols at the end?  

This is how it looks like:
![image](https://user-images.githubusercontent.com/62605534/106860480-1fa26f80-66cd-11eb-9472-eccb6ec82364.png)

Reply:
@SteveVaknin can you remove the symbols and merge please? :)

Reply:
@schosterbarak Sure 👌